src/Components/ArtworkGrid/ArtworkGrid.tsx

```diff
@@ -85,8 +85,11 @@ export class ArtworkGridContainer extends React.Component<
 
   private get _columnCount(): number[] {
     const columnCount = this.props.columnCount
-    // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-    return typeof columnCount === "number" ? [columnCount] : columnCount
+
+    if (typeof columnCount === "number") {
+      return [columnCount]
+    }
+    return Array.isArray(columnCount) ? columnCount : []
```

Comment:
Are the changes here auto generated?

Reply:
I was trying to remove the easy todo's that I've noticed and that is what I ended up with.